igor-web/src/main/groovy/com/netflix/spinnaker/igor/jenkins/service/JenkinsService.java

```diff
@@ -91,10 +90,7 @@ private String encode(String uri) {
   }
 
   public ProjectsList getProjects() {
```

Comment:
The `grouppKey` in here is a distinct name per jenkins host, so that we would stop requests to a bad jenkins master. I don't know that we actually get much benefit from tripping that circuit breaker when there are problems, but I wonder what still uses groupKey in here (and other spots that got de-hystrix'd)

Reply:
Just checked, `groupKey` isn't used anywhere else, I just failed to delete it.

I can add circuit breaking back into `JenkinsService` if we think it'd be useful. I suspect the only time it comes into play is when we take down a Jenkins master to upgrade - I think there's legitimacy there. WDYT?

Reply:
Added circuit breaking back in for jenkins.